charts/ratify/values.yaml

```diff
@@ -83,6 +83,7 @@ provider:
     cacheSizeMb: 256 # max size of the cache in MB
     ttl: 10s # cache ttl duration
     name: "" # state-store name for dapr cache, defaults to redis
+  enableMutation: true # enableMutation allows ratify to mutate image tag to image digest
```

Comment:
```suggestion
  enableMutation: true # enableMutation allows ratify to mutate image tag to image digest. It is highly recommended to enable mutation since the verified digest may be different from the one run
```

Reply:
Hi @akashsinghal Included the recommendation.